Source/Operations/RemoveIndexOperation.cs

```diff
@@ -4,11 +4,17 @@ namespace Rivet.Operations
 {
 	public sealed class RemoveIndexOperation : TableObjectOperation
 	{
-		public RemoveIndexOperation(string schemaName, string tableName, string name)
+		public RemoveIndexOperation(string schemaName, string tableName, string name, string[] columnName, bool unique)
 			: base(schemaName, tableName, name)
 		{
+			ColumnName = columnName;
+			Unique = unique;
 		}
 
+		public string[] ColumnName { get;  }

```

Comment:
Prefer IReadOnlyList/IReadOnlyCollection when practical.

Reply:
These are part of Rivet's API and each property needs to be modifiable for user's in their plug-ins.